templates/CRM/Member/Form/MembershipRenewal.tpl

```diff
@@ -303,5 +402,37 @@
         cj('#record-different-contact').hide();
       }
     }
+
+    function buildAmount( ) {
```

Comment:
Again, please respond either by adopting suggestion or explaining why it is not done/should not be done.